Makefile

```diff
@@ -315,6 +315,15 @@ azure-secrets:
 	oc create secret generic codecov-token --from-literal=upload=${CODECOV_UPLOAD_TOKEN} -o yaml --dry-run | oc apply -n azure -f -
 .PHONY: azure-secrets
 
+azure4-secrets:
```

Comment:
Please call this `azure-e2e-secrets` for now (azure4 is confusing).  @jim-minter since azure is camping on the convention that the rest of the e2e suites run, I would prefer if you guys can over time migrate from `azure-secrets` to `azure-aro-secrets` or similar.  The base namespace belongs to the core CI platform.